rspace/src/main/scala/coop/rchain/rspace/util/package.scala

```diff
@@ -111,4 +114,14 @@ package object util {
 
   val ordByteVector: Ordering[ByteVector] = (a: ByteVector, b: ByteVector) => veccmp(a, b)
 
+  val stringSerialize: Serialize[String] = new Serialize[String] {
+
+    def encode(a: String): ByteVector =
+      ByteVector.view(a.getBytes(StandardCharsets.UTF_8))
+
+    def decode(bytes: ByteVector): Either[Throwable, String] =
```

Comment:
this function never fails but the signature suggests it does fail on errors.

You should either change the type or report the decoding error properly